src/Mvc/Mvc/src/MvcServiceCollectionExtensions.cs

```diff
@@ -253,7 +253,7 @@ private static IMvcCoreBuilder AddControllersWithViewsCore(IServiceCollection se
     /// To add services for controllers with views call <see cref="AddControllersWithViews(IServiceCollection)"/>.
     /// </para>
     /// </remarks>
-    [RequiresUnreferencedCode("Razor Pages does not currently support native AOT.", Url = "https://aka.ms/aspnet/nativeaot")]
+    [RequiresUnreferencedCode("Razor Pages does not currently support trimming or native AOT,", Url = "https://aka.ms/aspnet/trimming")]
```

Comment:
Why the comma `,` at the end of the message?

Reply:
I fat fingered it.